lib/reporter/util.js

```diff
@@ -28,7 +28,22 @@ function hasFailures(modules) {
 
 function sanitizeOutput(output, delimiter, xml) {
   if (!output || output === '') return '';
-  return _(output.split(/[\r\n]+/)).filter(function (item) {
+  /* eslint-disable no-control-regex */
+  // ([valid-range])|. matches anything but only captures valid-range
+  var validUtf8 = new RegExp('(' + [
+    /[\x09\x0A\x0D\x20-\x7E]/,             // ASCII
+    /[\xC2-\xDF][\x80-\xBF]/,              // non-overlong 2-byte
+    /\xE0[\xA0-\xBF][\x80-\xBF]/,          // excluding overlongs
+    /[\xE1-\xEC\xEE\xEF][\x80-\xBF]{2}/,   // straight 3-byte
+    /\xED[\x80-\x9F][\x80-\xBF]/,          // excluding surrogates
+    /\xF0[\x90-\xBF][\x80-\xBF]{2}/,       // planes 1-3
+    /[\xF1-\xF3][\x80-\xBF]{3}/,           // planes 4-15
+    /\xF4[\x80-\x8F][\x80-\xBF]{2}/        // plane 16
+  ].map(function(r) {return r.source;}).join('|') + ')|.', 'g');
```

Comment:
@MylesBorins Looking at it again, I don't know why a multi-line string wouldn't be possible, but this doesn't seem to work.

This:

```js
var validUtf8v2 = new RegExp(
    '([\x09\x0A\x0D\x20-\x7E]|' +           // ASCII
    '[\xC2-\xDF][\x80-\xBF]|' +             // non-overlong 2-byte
    '\xE0[\xA0-\xBF][\x80-\xBF]|' +         // excluding overlongs
    '[\xE1-\xEC\xEE\xEF][\x80-\xBF]{2}|' +  // straight 3-byte
    '\xED[\x80-\x9F][\x80-\xBF]|' +         // excluding surrogates
    '\xF0[\x90-\xBF][\x80-\xBF]{2}|' +      // planes 1-3
    '[\xF1-\xF3][\x80-\xBF]{3}|' +          // planes 4-15
    '\xF4[\x80-\x8F][\x80-\xBF]{2})|.'      // plane 16
    , 'g');
```

should be the same as this:

```js
var validUtf8 = new RegExp('(' + [
  /[\x09\x0A\x0D\x20-\x7E]/,             // ASCII
  /[\xC2-\xDF][\x80-\xBF]/,              // non-overlong 2-byte
  /\xE0[\xA0-\xBF][\x80-\xBF]/,          // excluding overlongs
  /[\xE1-\xEC\xEE\xEF][\x80-\xBF]{2}/,   // straight 3-byte
  /\xED[\x80-\x9F][\x80-\xBF]/,          // excluding surrogates
  /\xF0[\x90-\xBF][\x80-\xBF]{2}/,       // planes 1-3
  /[\xF1-\xF3][\x80-\xBF]{3}/,           // planes 4-15
  /\xF4[\x80-\x8F][\x80-\xBF]{2}/        // plane 16
 ].map(function(r) {return r.source;}).join('|') + ')|.', 'g');
```

but it doesn't seem to be (as far as I can tell).